scripts/state_backup.sh

```
@@ -40,22 +63,21 @@ abci_import_state() {
   docker run --rm -v "${volume}:/data" busybox:1.36 sh -c 'rm -rf /data/*'
   docker run --rm -v "${volume}:/data" -v "$dir:/in:ro" busybox:1.36 sh -c "cd /data && tar xzp -f /in/$file"
 }
-abci_transfer_help() { printf 'Move the archive manually, for example:\n  scp drive_abci_state_<timestamp>.tar.gz user@remote:/path\nThen on the target host run:\n  scripts/transfer_drive_abci_state.sh import /path/drive_abci_state_<timestamp>.tar.gz [config] --component abci\n'; }
 tenderdash_resolve_volume() {
   local cfg=$1
-  local project=$(${DASHMATE_BIN} config envs --config "$cfg" | awk -F= '$1=="COMPOSE_PROJECT_NAME"{print $2}')
+  local project=$("${DASHMATE_CMD[@]}" config envs --config "$cfg" | awk -F= '$1=="COMPOSE_PROJECT_NAME"{print $2}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)